backend/server/app.py

```
@@ -210,3 +261,296 @@ async def health_check():
+            "r2_score": r2_score,
+            "calibrated_surface": None,  # Would contain actual calibrated surface data
+            "surface_id": surface_id,
+            "asset_id": asset_id,
+            "timestamp": datetime.now().isoformat(),
+            "calibration_method": "mock_cnn"  # or "optimization"
+        }
+        
+        return JSONResponse(content=calibration_data)
+    except Exception as e:
+        logging.error(f"Error getting calibration data: {str(e)}")
+        raise HTTPException(status_code=500, detail=f"Error getting calibration data: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)